src/runtime/composables/useScript.ts

```diff
@@ -25,7 +25,7 @@ export function useScript<T extends Record<string | symbol, any>>(input: UseScri
     if (!nuxtApp._scripts?.[id]) {
       performance?.mark?.('mark_feature_usage', {
         detail: {
-          feature: `nuxt-scripts:${id}`,
+          feature: `nuxt-third-parties-${id}`,
```

Comment:
I'd prefer if the non-TPC scripts used the old key name.

So could we add a config key on NuxtUseScriptOption `performanceMarkFeature` which can be added as deprecated, then we provide the full key via the TPC scripts

```
feature: options?.performanceMarkFeature || ? `nuxt-scripts:${id}`,
```

Reply:
i'm good with this  :+1:

Reply:
as deprecated ? Maybe as internal if you don't want it to be public